sdk/include/reactos/dbgbitmap.h

```diff
@@ -0,0 +1,359 @@
+    _In_ ULONG FromIndex,
+    _Out_ PULONG StartingRunIndex
+);
+
+static inline
+ULONG
+RtlFindNextForwardRunSetDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader,
+    _In_ ULONG FromIndex,
+    _Out_ PULONG StartingRunIndex)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    return RtlFindNextForwardRunSet(&BitMapHeader->BitMap, FromIndex, StartingRunIndex);
+}
+
+static inline
+ULONG
+RtlFindSetBitsDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader,
+    _In_ ULONG NumberToFind,
+    _In_ ULONG HintIndex)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    return RtlFindSetBits(&BitMapHeader->BitMap, NumberToFind, HintIndex);
+}
+
+static inline
+ULONG
+RtlFindSetBitsAndClearDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader,
+    _In_ ULONG NumberToFind,
+    _In_ ULONG HintIndex)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    return RtlFindSetBitsAndClear(&BitMapHeader->BitMap, NumberToFind, HintIndex);
+}
+
+static inline
+ULONG
+RtlNumberOfClearBitsDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    return RtlNumberOfClearBits(&BitMapHeader->BitMap);
+}
+
+static inline
+ULONG
+RtlNumberOfSetBitsDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    return RtlNumberOfSetBits(&BitMapHeader->BitMap);
+}
+
+static inline
+VOID
+RtlSetBitDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader,
+    _In_range_(<, BitMapHeader->SizeOfBitMap) ULONG BitNumber)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    if (RtlCheckBit(&BitMapHeader->BitMap, BitNumber) == FALSE)
+    {
+        BitMapHeader->NumberOfSetBits++;
+    }
+    RtlSetBit(&BitMapHeader->BitMap, BitNumber);
+    BitMapHeader->BitmapHash = RtlComputeBitmapHashDbg(BitMapHeader);
+}
+
+static inline
+VOID
+RtlSetBitsDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader,
+    _In_range_(0, BitMapHeader->SizeOfBitMap - NumberToSet) ULONG StartingIndex,
+    _In_range_(0, BitMapHeader->SizeOfBitMap - StartingIndex) ULONG NumberToSet)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    RtlSetBits(&BitMapHeader->BitMap, StartingIndex, NumberToSet);
+    BitMapHeader->NumberOfSetBits = RtlNumberOfSetBits(&BitMapHeader->BitMap);
+    BitMapHeader->BitmapHash = RtlComputeBitmapHashDbg(BitMapHeader);
+}
+
+static inline
+VOID
+RtlSetAllBitsDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    RtlSetAllBits(&BitMapHeader->BitMap);
+    BitMapHeader->NumberOfSetBits = BitMapHeader->BitMap.SizeOfBitMap;
+    BitMapHeader->BitmapHash = RtlComputeBitmapHashDbg(BitMapHeader);
+}
+
+_Must_inspect_result_
+static inline
+BOOLEAN
+RtlTestBitDbg(
+    _In_ PRTL_BITMAP_DBG BitMapHeader,
+    _In_range_(<, BitMapHeader->SizeOfBitMap) ULONG BitNumber)
+{
+    RtlValidateBitmapDbg(BitMapHeader);
+    return RtlTestBit(&BitMapHeader->BitMap, BitNumber);
+}
+
+_Must_inspect_result_
+static inline
+BOOLEAN
+RtlCheckBitDbg(
+  _In_ PRTL_BITMAP_DBG BitMapHeader,
+  _In_range_(<, BitMapHeader->SizeOfBitMap) ULONG BitPosition)
```

Comment:
```suggestion
    _In_ PRTL_BITMAP_DBG BitMapHeader,
    _In_range_(<, BitMapHeader->SizeOfBitMap) ULONG BitPosition)
```
(making it consistent with the others)

Reply:
done